build/update_ext_version.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import datetime
+import json
+import pathlib
+
+EXT_ROOT = pathlib.Path(__file__).parent.parent
+PACKAGE_JSON_PATH = EXT_ROOT / "package.json"
+
+
+def build_arg_parse():
+    """Builds arguments parser for the following arguments.
+        --release: Validates release version number.
+        --build-id: Custom <micro> version for the build.
+        --for-publishing: Ensures that the version is parseable by marketplace.
```

Comment:
No need to complicate the docstring when `-h` and the help per arg is going to track what flags are supported.

```suggestion
    """Builds the arguments parser."""
```